data/config/includes/DefaultStyles.include.xml

```diff
@@ -143,16 +143,30 @@
     <RenderMethod>nearest,bilinear,point,barb</RenderMethod>
     <StandardNames standard_name="*"/>
   </Style>
-  
+
   <Style name="autofeatures">
     <Legend fixedclasses="true" tickround="1">elevation</Legend>
     <Point pointstyle="point" textcolor="#111111"/>
     <Vector vectorstyle="barb" linecolor="#0000FF" />
-    <FeatureInterval match=".*" bgcolor="#CCCCFF" fillcolor="#000080FF"  bordercolor="#000000" borderwidth="2"/> <!-- Used in CImgRenderPolylines -->
+    <FeatureInterval match=".*"
```

Comment:
This is for autowms?

Reply:
Good question. It is for autowms, but what should be shown iby autowms. You don't know in advance what to show here.

Any suggestion?

Reply:
I guess it is Ok to do this, but when should autofeatures be added to the stylelist?

Reply:
Not sure what to do with this, as there is no uiniversal default for matchid

Reply:
I suggest to leave this as is and decide how to do this later, after we've had some more experience